setup/README.md

```diff
@@ -0,0 +1,87 @@
+# Orka VM Setup Script Usage
+
+## This script automates the configuration and setup of macOS Orka VMs for optimal performance. It configures system settings, installs required tools, and prepares the VM for use with Orka
+
+### System requirements
+
+- macOS IPSW file (Tahoe, Sequoia, or Sonoma)
+- Admin/sudo privileges on the VM
+- Internet connection for downloading packages
+- Terminal access (via SSH, Screen Sharing, or direct console)
+
+### Before running
+
+- Ensure you have administrator access to the VM
+- Close any important applications (they will be terminated during cleanup)
+- Save any work in progress (the VM will reboot automatically)
+
+### Installation
+
+#### Download via GitHub
+
+- `bash /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/macstadium/orka-images/setup/setup.sh)"`
```

Comment:
This still does not render correctly. I would suggest previewing the render before pushing the changes for the readme.

<img width="1041" height="137" alt="Screenshot 2025-10-02 at 2 30 10 PM" src="https://github.com/user-attachments/assets/b4e4f235-95e0-4d5c-b753-0f5a7f291b82" />

The correct way to write this (if you want syntax highlighting for `bash`) is to use triple backticks.

Like this:

````
```bash
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/macstadium/orka-images/setup/setup.sh)"
```
````

This will render as:

```bash
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/macstadium/orka-images/setup/setup.sh)"
```

Reply:
thanks, pushed a new commit